### ISSUE TITLE
Tensorflow 2.0?

### ISSUE THREAD

**Issue by Jmkernes** [poolio/unrolled_gan#8]
Is there any way to migrate this code to TensorFlow 2.0+? It seems like in V2 we no longer have access to the graph_replace function.

Also, in V2's keras implementation, the optimizer.get_updates() method only accepts 2 arguments as opposed to 3 (loss, variables), and appears to be virtual, since it breaks with a "no gradients exist error" when trying to call it.
